lms/djangoapps/grades/tests/test_course_grade_factory.py

```diff
@@ -54,57 +46,6 @@ def test_course_grade_no_access(self):
         grade = CourseGradeFactory().read(self.request.user, invisible_course)
         assert grade.percent == 0
 
-    @patch.dict(settings.FEATURES, {'PERSISTENT_GRADES_ENABLED_FOR_ALL_TESTS': False})
-    @ddt.data(
```

Comment:
These tests are removed because the value `PERSISTENT_GRADES_ENABLED_FOR_ALL_TESTS` should be `True` all the time now, right? Are there tests within this file covers the case where `PERSISTENT_GRADES_ENABLED_FOR_ALL_TESTS` is `True`? I am mindful of test coverage percentage.

Reply:
Yes, these tests are removed because the value `PERSISTENT_GRADES_ENABLED_FOR_ALL_TESTS` should be `True` all the time now.
`PERSISTENT_GRADES_ENABLED_FOR_ALL_TESTS` feature removed from platform at all. This feature used in [PersistentGradesEnabledFlag feature_enabled](https://github.com/openedx/edx-platform/pull/30978/files#diff-61855a03b2acfd3689924fa13918008b4dc22dee0089f75256568bb8fe1d2ebcL43) function. 
I believe [test_read_and_update](https://github.com/openedx/edx-platform/pull/30978/files#diff-d009728fd48d9c7ca695f14cb0e3f71c712aab9ba68d8b7390a36bb40d210d6bL108) covers this case.